### PR TITLE
Put arraybuffer contents in blobs while rehydrating container

### PR DESCRIPTION
**Pull request by jatgarg**

### CODE REVIEW

File: packages/loader/container-loader/src/utils.ts [hunk 1]
```diff
@@ -74,7 +74,7 @@ function convertSummaryToSnapshotWithEmbeddedBlobContents(
                 const blobId = uuid();
                 treeNode.blobs[key] = blobId;
                 const contentBuffer = typeof summaryObject.content === "string" ?
-                    stringToBuffer(summaryObject.content, "utf8") : summaryObject.content;
+                    stringToBuffer(summaryObject.content, "utf8") : summaryObject.content.buffer;
```
Comment (vladsud) [inline packages/loader/container-loader/src/utils.ts:77]: This is not fully correct, as Uint8Array might be a partial view into its underlying array buffer.
You recently did slicing in ODSP driver for similar flow. We should create a helper function that does the right stuff (usage of full buffer or slice depending on situation).
Better fix I believe would be to move use one type, across all subsystems. I think Uint8Array is better, not sure why we chose ArrayBuffer in the first place

Comment (jatgarg) [reply packages/loader/container-loader/src/utils.ts]: Made the changes. I will open an issue to make changes from ArrayBuffer to Uint8 at other places.

File: packages/loader/driver-utils/src/utils.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export const uint8ArrayToArrayBuffer = (array: Uint8Array): ArrayBuffer =>
+    array.buffer.slice(array.byteOffset, array.byteOffset + array.byteLength);
```
Comment (vladsud) [inline packages/loader/driver-utils/src/utils.ts:7]: Could you please open an issue to move this to common-utils? Does not need to block this PR, but we should move it out of this layer.
As part of move, I'd be great to generilize code in IsoBuffer.from(), where we do have 0x000 & 0x001 asserts today.

Also can it not do slice when array.byteOffset === 0 && byteLength === array.value.buffer.byteLength?

**Review by vladsud — APPROVED** [review]

File: packages/loader/driver-utils/src/utils.ts (hunk 2 above)
Comment (jatgarg) [reply packages/loader/driver-utils/src/utils.ts:7]: https://github.com/microsoft/FluidFramework/pull/7013

Comment (jatgarg) [reply packages/loader/driver-utils/src/utils.ts:7]: Created Pr to actually add it in common-utils. I will remove that from here(Driver-utils).

**Pull request closed without merging**